pages/common/git-repl.md

```diff
@@ -0,0 +1,21 @@
+# git repl
+
+> Open a Git shell-like command line.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-repl>.
+
+- Open a Git shell-like command line:
```

Comment:
I think `open` is not correct
```suggestion
- Start an interactive shell with Git subcommands as top level commands:
```

Reply:
That sounds a little man pagey. Most wont know what a top level command line is

Reply:
However the beginning is a good change

Reply:
How about `Start an interactive Git shell`?

Reply:
This is what I would like the most.
```suggestion
- Start an interactive shell where you can use Git subcommands as though they were normal commands.
```
But it's got `your` in 🤔

Reply:
Like this?

```suggestion
- Start an interactive Git shell:
```

Reply:
@ bl-ue, I think `Git shell` implies that you can use subcommands as top level commands. It just seems intuitive.

Reply:
I agree

Reply:
Must be since you both think it.  Personally, I _might_ be confused by that, but since I've know what it means for a long time I can't really tell if I'd really be.  We'll see though if users find it strange and then have enough interest to inquire as to why we wrote it 🙂

Reply:
So which suggestion are we applying?

Reply:
I have applied my suggestion